src/clj/vanilla/power_data_service.clj

```diff
@@ -42,6 +42,24 @@
 
 
 (comment
+
```

Comment:
can you please add some comments here to explain what this code does and when another developer might want to poke at it? (I need to do this more myself)

Reply:
removed it, it wasnt even related to power service, it was leftover from me needing a clj comment block to play with autosizer stuff in.